restcomm/restcomm.telephony/src/main/java/org/restcomm/connect/telephony/CallManager.java

```diff
@@ -388,6 +388,8 @@ public CallManager(final Configuration configuration, final ServletContext conte
             }
         }
 
+        this.uriUtils = RestcommConnectServiceProvider.getInstance().uriUtils();
```

Comment:
I think it's pretty bad for testability to retrieve dependencies in a static way like this. Can we expose this via constructor instead?